virt/node_maintenance/virt-using-skip-node.adoc

```diff
@@ -0,0 +1,13 @@
+[id="virt-using-skip-node"]
+= Using skip node annotation
+include::modules/virt-document-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: virt-using-skip-node
+
+toc::[]
+
```

Comment:
Assemblies need introductions (though they sometimes slip through the cracks without them). I'd suggest one or two sentences that summarize why you'd want to use skip node annotation.